### ISSUE TITLE
daily report

### ISSUE THREAD

**Issue by buckbowl** [opensourcepos/opensourcepos#2671]
Información de la Instalación OSPOS: | 3.3.1 - c786d4
es
» GD: Enabled ✓
» BC Math: Enabled ✓
» INTL: Enabled ✓
» OpenSSL: Enabled ✓
» MBString: Enabled ✓
» Curl: Enabled ✓
.Browser: Chrome.Server Software: Apache.PHP Version: 7.3.11.DB Version: 5.7.28.Server Port: 80.OS: Linux phoenix.viewen.com 3.10.0-962.3.2.lve1.5.26.7.el7.x86_64 #1 SMP Wed Oct 2 07:53:12 EDT 2019 x86_64
» [application/logs:] - 0755 | Writable ✓ 
» [public/uploads:] - 0755 | Writable ✓ 
» [public/uploads/item_pics:] - 0755 | Writable ✓ 

**Hi, I want to see if it is possible for someone to help me modify the daily sales report file.**

![ventas d 1](https://user-images.githubusercontent.com/59344753/72007945-61d25680-3218-11ea-9834-f8e662043d45.jpg)

So that in this circle the total sales data appear.
Date:
Stock location: #
Cash Sales: $
Credit Card Sales : $
Total Sales: $
**And that ONLY this data is printed on a ticket printer**
![ventas d 2](https://user-images.githubusercontent.com/59344753/72008065-a65df200-3218-11ea-8fd6-d4ae8a9c10bf.jpg)

Thanks in advance for your help, it is very important for my business ....

**Comment by WebShells** [comment]
You can add media print none for daily sales table, this will not print the report, will only print header and summary data... The summary u want to show is already half there... U should add stock # only...

**Comment by buckbowl** [comment]
Thank you.
What file is it and in which subdirectory is it?

**Comment by buckbowl** [comment]
> You can add media print none for daily sales table, this will not print the report, will only print header and summary data... The summary u want to show is already half there... U should add stock # only...


Thank you.
What file is it and in which subdirectory is it?

**Comment by buckbowl** [comment]
Hi, could someone help me do that even without giving a user permission to view sales reports. 
He YES can see the daily sales report? Thank you
![day sales](https://user-images.githubusercontent.com/59344753/72212202-4b88fc80-349d-11ea-94ed-4dbbefbdc717.jpg)

**Comment by buckbowl** [comment]
> You can add media print none for daily sales table, this will not print the report, will only print header and summary data... The summary u want to show is already half there... U should add stock # only...


We definitely need a daily sale closing for each of the stocks. I have 3 stores in different cities logically managed by different cashiers, so at the end of their day they need to know how much they sold (in cash or cards) to know how much money they have to have. I thought I could solve it with the daily sales report but it gives the result of ALL the stocks. I wish I could evaluate the possibility of creating a daily sales closing for each stock and that the cashiers can see it without needing to have permissions to see reports in general.
PLEASE HELPMY..

**Comment by odiea** [comment]
So you want them to be able to give to you a daily Sales report from a Daily Sales Report. My understanding is they need to add up all the Sales that  are in their Register Then You compare it to what the Daily Sales Report is. That's the purpose of the Cashup feature .

**Comment by buckbowl** [comment]
> So you want them to be able to give to you a daily Sales report from a Daily Sales Report. My understanding is they need to add up all the Sales that are in their Register Then You compare it to what the Daily Sales Report is. That's the purpose of the Cashup feature .


CASHUP,  doesn't work, because it combines sales in all stores (locations)..

**Comment by odiea** [comment]
So I assume then that you are using 1 database for the 3 locations and have given the employees permission only for a certain location.

**Comment by daN4cat** [comment]
I'm afraid the cashup doesn't work by stock location.
That is what it is.

**Comment by buckbowl** [comment]
> So I assume then that you are using 1 database for the 3 locations and have given the employees permission only for a certain location.


Exactly, I only use 1 system for the three locations, previously I used BLUEPOS and there if there are dayli reports for each of the stores (stocks-locations) but it is an extremely expensive system although very complete. I think what I will do to avoid this is to use subdomains for my 3 different stores and for the closing of the day we will use cashup. Could you tell me how I can print the cashup form?

**Comment by mckaygerhard** [comment]
hi @buckbowl  htere's some documentation in the wiki how to started to make changes in ospos.. and only a hint! each report are take from dinamic tables not directly from data tables.. here used so many temp tables in each wquery to make more easy the filtering and management of data by the ajax calls!

**Comment by MichaelJay1232** [comment]
Hello, all my reports are not showing

**Comment by MichaelJay1232** [comment]
Please, its a web based, make transactions but can't see them in daily report or any other report

**Comment by azhar017** [comment]
@buckbowl  have you solved the issue yet?

**Comment by azhar017** [comment]
Daily Sales show all sales from all stock location. Is there a way to show separately in daily sale base on user permission on stock locations.

**Comment by jekkos** [comment]
This is a vague customization request. Please use GitHub Discussions with specific requirements for daily sales report format. OSPOS has various sales reports available.
